AirCasting/Notes/AddNote/AddNoteViewModel.swift

```diff
@@ -9,14 +9,14 @@ class AddNoteViewModel: ObservableObject {
     private let notesHandler: NotesHandler
     private let exitRoute: () -> Void
     
-    init(sessionUUID: SessionUUID, exitRoute: @escaping () -> Void) {
+    init(sessionUUID: SessionUUID, withLocation: Bool, exitRoute: @escaping () -> Void) {
```

Comment:
Doń;t you want to use the info about location: `withLocation`? Now I think it is not used, as before you were providing fake location info when this `var` was `true`